library/Requests.php

```diff
@@ -572,6 +572,9 @@ protected static function set_defaults(&$url, &$headers, &$data, &$type, &$optio
 				$options['data_format'] = 'body';
 			}
 		}
+
```

Comment:
`$type` var should be changed to uppercase at the start of this function since `$type` var is used to define data format.
Let me know if you want me to create PR or update it.